### ISSUE TITLE
Feature Request: Transition modifier for RGB/Value layers

### ISSUE THREAD

**Issue by lackofbindings** [ucupumar/ucupaint#219]
The one thing that is missing from my traditional Photoshop-based workflow is what would essentially be the Transition modifier that is available for normals, but for color/grayscale layers. Like Photoshop's Outer/Inner Glow, or Blender compositor's Dilate/Erode node. Ideally it might also have a curve input to control the falloff (linear vs exponential, etc.).

I do suspect this may not be possible since in all my research I cannot find any shader node setup that can achieve this effect, it seems only possible via the compositor.

**Comment by ucupumar** [comment]
Yes, the advanced effect like that is not possible yet in blender unless it can do texture-based nodes. The current transition effect on normal channel is also a bit of a hack.
